website/app/components/doc/related-components/index.hbs

```diff
@@ -6,4 +6,18 @@
 <hr class="doc-markdown-hr" />
 
 <h2 class="doc-text-h2">Related</h2>
-<Doc::Cards::Deck @cols="2" @layout="horizontal" @cards={{@cards}} />
\ No newline at end of file
+
+<Doc::Cards::Deck @cols="2">
+  {{#each @cards as |card|}}
+    <Doc::Cards::Card
+      @image={{card.image}}
+      @title={{card.title}}
+      @caption={{card.caption}}
+      @route={{card.route}}
+      @model={{card.model}}
+      @href={{card.href}}
+      @layout="horizontal"
+      {{doc-track-event eventName="Related - {{@componentTitle}} - {{card.title}}"}}
```

Comment:
@zamoore this isn't actually working. I think you need to `concat` these values into this string or something similar

<img width="729" alt="Screenshot 2024-06-13 at 09 42 39" src="https://github.com/hashicorp/design-system/assets/1672302/98cf52b5-1b8a-4da6-ae4e-d396292a2761">

Reply:
Addressed [here](https://github.com/hashicorp/design-system/pull/2165)